src/app/movies/movies.component.html

```diff
@@ -15,7 +15,7 @@
     </mat-menu>
   </div>
 </div>
-<div class="filmNew" *ngFor="let movie of movies">{{ movie.id }}
+<div class="filmNew" *ngFor="let movie of movies">{{movie}}
```

Comment:
proponuję się trzymać jednej wersji tzn klasa movie a nie newFilm (movie bo to nie będze miało znaczeni czy on jest nowy czy nie nowy, wszystkie tu wyświetlisz)